applications/StructuralMechanicsApplication/tests/test_dynamic_schemes.py

```diff
@@ -46,22 +46,25 @@ def _base_spring_test_pseudo_static_scheme(self, current_model, scheme_name = "p
         current_disp = 0.0
         current_vel = init_velocity
         # Solve the problem
-        while(time <= end_time):
+        while time <= end_time:
             time = time + dt
             step = step + 1
             mp.CloneTimeStep(time)
             mp.ProcessInfo[KratosMultiphysics.STEP] = step
 
             self.strategy.Solve()
 
-            current_analytical_displacement_y = current_disp + current_vel * dt - current_vel * beta * mass * dt/2.0
+            current_analytical_displacement_y = current_disp + current_vel * dt * (1.0 - 5.0 * beta * mass)
             current_disp = current_analytical_displacement_y
-            current_analytical_velocity_y = current_vel - current_vel * beta * mass
+            current_analytical_velocity_y = current_vel
             current_vel = current_analytical_velocity_y
 
-            ## ASSERT
-            self.assertAlmostEqual(node.GetSolutionStepValue(KratosMultiphysics.DISPLACEMENT_Y,0), current_analytical_displacement_y, delta=1e-3)
-            self.assertAlmostEqual(node.GetSolutionStepValue(KratosMultiphysics.VELOCITY_Y,0), current_analytical_velocity_y, delta=1e-3)
+            computed_disp = node.GetSolutionStepValue(KratosMultiphysics.DISPLACEMENT_Y,0)
+            computed_vel = node.GetSolutionStepValue(KratosMultiphysics.VELOCITY_Y,0)
+
+            ### ASSERT
+            self.assertAlmostEqual(computed_disp, current_analytical_displacement_y, delta=1e-3)
```

Comment:
i think this should be way more precise than 1e-3 !!!!

Reply:
there original test was like that... any clues on why?

Reply:
Huum, let me check

Reply:
The original test was already 1.0e-3